app/classifier/tasks/dropdown/dropdown-dialog.cjsx

```diff
@@ -77,7 +85,7 @@ DropdownDialog = React.createClass
     @setState editSelect: select
 
   onClickAddOption: ->
-    @addOption(@refs.optionInput.value)
+    @addOption @refs.optionInput.value
     @refs.optionInput.value = ''
```

Comment:
Sorry for the delay. I think this is fine here. Manipulating the DOM directly in a React component is very unusual, I'd avoid it in the future.